packages/devtools/lib/src/timeline/frames_bar_chart.dart

```diff
@@ -23,6 +31,14 @@ class FramesBarChart extends CoreElement {
       ..paddingTop = '${topPadding}px';
 
     timelineController.onFrameAdded.listen((TimelineFrame frame) {
+      if (element.children.isEmpty) {
+        frameUIgraph = PlotlyDivGraph(this, frame, false); // Process chunks
+        add(frameUIgraph);
+      }
+
+      print(">>>>onFrameAdded Event");
```

Comment:
remove all print statements before commit.

Reply:
I've only kept the HACK to detect bad data when the Trace error happens.  That should be removed. @kenzieschmoll do we know what the problem is and how to fix.

Reply:
If you're talking about https://github.com/flutter/devtools/issues/301, this is something I need to ask Chinmay about. In theory we shouldn't be getting multiple frame start/end events with the same frame id, but when that happens, we see this logged to console.